test/minitest/xmfun/util/url_parser_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'mini_test_helper'
+
+module Xmfun
+	module Util
+		class TestUrlParser < Minitest::Test
+			def setup
+				@urlParser = Xmfun::Util::UrlParser
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Use snake_case for variable names.